scripts/stop_part.py

```diff
@@ -0,0 +1,51 @@
+#! /usr/bin/env python
+# -*- mode: python; coding: utf-8 -*-
+# Copyright 2017 the HERA Collaboration
+# Licensed under the 2-clause BSD license.
+
+"""
+Script to add a general connection to the database.
+"""
+
+from __future__ import absolute_import, division, print_function
+
+from hera_mc import mc, cm_utils, part_connect
+import sys
+import copy
+
+
+def query_args(args):
+    """
+    Gets information from user
+    """
+    if args.part is None:
+        args.part = raw_input("Part number:  ")
+    if args.rev is None:
+        args.rev = raw_input("Revision:  ")
+    if args.date == 'now':
```

Comment:
Thanks - I've addressed your comments.  The --date now thing is somewhat tricky since date is "universal", but I think I've described it in the header comment.